homeassistant/components/camera/proxy.py

```diff
@@ -56,32 +56,21 @@
     async_add_devices([ProxyCamera(hass, config)])
 
 
-async def _read_frame(req):
-    """Read a single frame from an MJPEG stream."""
-    # based on https://gist.github.com/russss/1143799
-    import cgi
-    # Read in HTTP headers:
+# This routine is adapted from the one in mjpeg.py.  They should probably
```

Comment:
Let's just merge it together? Put it in a file called `camera/_util.py`

Reply:
I don't have any way to test the MJPEG camera if I make the change.  I can make it anyway, but my track record on getting the asyc stuff right isn't exactly stellar.

Reply:
okay, let's not extract it this time.